psi4/driver/qcdb/libmintsbasissetparser.py

```diff
@@ -247,6 +247,8 @@ def parse(self, symbol, dataset):
 
                                 if len(shell_type) == 1:
                                     am = shell_to_am[shell_type[0]]
```

Comment:
```suggestion
                                if len(shell_type) == 1 or (len(shell_type) >= 3 and shell_type[:2] == 'L='):
                                    if len(shell_type) == 1:
                                        am = shell_to_am[shell_type[0]]
                                    else:
                                        am = int(shell_type[2:])
```
Would this do away with the new block below?

Reply:
Resolved differently